examples/branin/restore_state.py

```diff
@@ -23,12 +23,14 @@ def main():
         'deterministic' : True,
         'output_dir' : 'restore_me'}
     original_scenario = Scenario(orig_scen_dict)
-    smac = SMAC(scenario=original_scenario)
+    smac = SMAC(scenario=original_scenario,
+                run_id=1)
     smac.optimize()
 
     print("\n########## BUDGET EXHAUSTED! Restoring optimization: ##########\n")
 
-    # Now the output is in the folder 'restore_me'
+    # Now the output is in the folder 'restore_me/run_1'
+    old_output_dir = os.path.join(original_scenario.output_dir, 'run_1')
```

Comment:
it doesn't has to be in `run_1`; could also be `run_20`, right?

Reply:
Yep. Added a comment.